src/components/LowerLayout/Footer/index.js

```diff
@@ -1,33 +1,31 @@
 import React from 'react';
-import openUpLogo from '../../../images/OpenUp-logo.png'
+import openUpLogo from '../../../images/OpenUp-logo.png';
 
 import classes from './style.module.css';
 
-const Footer = () => {
-  return (
+const Footer = () => (
```

Comment:
Perhaps break this up into smaller variables, currently it's a bit unreadable. So you would ideally have something like:
```
<div>
  {logo}
  <div>
    {contact}
    {disclaimers}
  </div>
</div>
```

Reply:
Refactored :)